docs/scripts_md/MakeArchiveLocationRelative.md

```diff
@@ -0,0 +1,53 @@
+# NAME
+
+MakeArchiveLocationRelative.pl -- Removes the root directory from the
+`ArchiveLocation` field in the `tarchive` table to make the path to the
+tarchive relative.
+
+# SYNOPSIS
+
+perl MakeArchiveLocationRelative.pl `[options]`
+
+Available option is:
+
+\-profile: name of the config file in ../dicom-archive/.loris\_mri
+
+# DESCRIPTION
+
+This script will remove the root directory from the ArchiveLocation field
+in the tarchive table to make path to the tarchive relative. This should
```

Comment:
here too for the `tarchive`

Reply:
I would re-word the sentence to say: "to make the .tar path a relative one".

Reply:
Nice! Done!